101-private-endpoint-webapp/azuredeploy.json

```diff
@@ -0,0 +1,224 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "virtualNetwork_name": {
+            "type": "String"
+        },
+        "serverFarm_name": {
+            "type": "String"
+        },
+        "site_name": {
+            "type": "String"
+        },
+        "virtualNetwork_CIDR": {
+            "type": "String",
+            "defaultValue": "10.200.0.0/16"
+        },
+        "subnet1_name": {
```

Comment:
add defaultValues for all the params where it makes sense (resource names, skus)

Reply:
already done for non unique value